packages/react-utilities/src/compose/types.ts

```diff
@@ -65,17 +65,17 @@ export type RequiredProps<T, K extends keyof T> = Omit<T, K> & { [P in K]-?: T[P
  * * Ensures the specified ShorthandProps are of type ObjectShorthandProps<T>
  * * Marks the given DefaultedProps as required (-?)
  *
- * @param Props - The component's Props type
- * @param RefType - The type of the state.ref property; e.g. `React.Ref<HTMLElement>`
- * @param ShorthandProps - The keys of Props that correspond to ShorthandProps
- * @param DefaultedProps - The keys of Props that will always have a default value provided
+ * @template Props - The component's Props type
+ * @template RefType - The type of the state.ref property; e.g. `React.Ref<HTMLElement>`
+ * @template ShorthandPropNames - The keys of Props that correspond to ShorthandProps
```

Comment:
This was flagged because the `ShorthandProps` type parameter shadowed the type `ShorthandProps` defined earlier in the file. Open to other suggestions for new names. (Also type params should be documented with `@template` not `@param`.)

@behowell FYI